news/2021-09-15-new-featured-artist-rejection.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "New Featured Artist: rejection"
+date: 2021-09-15 09:00:00 +0000
+---
+
+Treat yourself to the latest addition to our Featured Artist catalogue: **rejection**!
+
+![](https://assets.ppy.sh/artists/199/header.jpg)
+
+Exemplifying the core of doujin rhythm game music, **rejection** is a DJ/producer whose songs span from instrumental masterpieces to vocal works featuring notable singers in the doujin scene like *Such*, [*Aitsuki Nakuru*](https://osu.ppy.sh/beatmaps/artists/127), and *mami*. No matter what you're looking for, **rejection** has you covered.
+
+**14** songs are up for grabs from [**rejection**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/199), most of which come with a game-size 2:00 version supplied by the artist specifically for osu!. Thanks for being cool, **rejection**.
```

Comment:
is this what is meant?

```suggestion
**14** songs are up for grabs from [**rejection**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/199), most of which come with a game-size 2.0 version supplied by the artist specifically for osu!. Thanks for being cool, **rejection**.
```

Reply:
or 
```suggestion
**14** songs are up for grabs from [**rejection**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/199), most of which come with a game-size 2-minute version supplied by the artist specifically for osu!. Thanks for being cool, **rejection**.
```

Reply:
second is the only valid reading of `2:00`, but `2 minute` or `two-minute` would read better.

Reply:
agree. the only reason one might think twice (or for me being dumb) is because "version" is tacked on and the former may sound plausible